bin/node/runtime/src/lib.rs

```diff
@@ -1575,6 +1575,9 @@ parameter_types! {
 	);
 	pub const LLMMaxTotalLocked: Balance = 100_000 * GRAINS_IN_LLM;
 	pub const LLMMinimumTransfer: Balance = 10 * GRAINS_IN_LLM;
+	pub const BridgeMinimumFee: Balance = 1 * DOLLARS;
```

Comment:
minium fee in LLD is too high, 1 LLD should be around 100 simple transactions. I think for this 0.1 min is ok,

Reply:
Done, lowered to 0.10 LLD.